ccmlib/scylla_node.py

```diff
@@ -1469,7 +1469,13 @@ def do_invoke(sstables):
                 return (stdout, stderr)
             common_args = [scylla_path, "sstable", command] + additional_args
             env = self._get_environ()
-            res = subprocess.run(common_args + sstables, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=text, check=True, env=env)
+            res = subprocess.run(common_args + sstables, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=text, check=False, env=env)
+            if res.returncode:
+                msg = f"{common_args + sstables} failed:\n"
+                for l in res.stderr.split('\n'):
+                    msg += f"> {l}\n"
+                self.debug(msg)
+                res.check_returncode()
```

Comment:
Mayne use `ToolError`, like in nodetool and c-s calls ? it raise with the stdout and stderr

you can even use `handle_external_tool_process` directly as in `stress()`, just need to switch to `subprocess.Popen` instead of `subprocess.run`

Reply:
Thanks.  Done in next version.